Dockerfile

```
@@ -7,17 +7,16 @@
 # We don't use alpine or -slim, because pandas is a pain to build on them.
 FROM python:3.11
 
-RUN pip install --no-cache-dir poetry==1.8.2
+RUN pip install --no-cache-dir uv==0.7.9
```

https://docs.astral.sh/uv/guides/integration/docker/#installing-uv
```suggestion
COPY --from=ghcr.io/astral-sh/uv:0.7.20 /uv /uvx /bin/
```